app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -3956,6 +3956,24 @@ class DatabaseMigrationAndroidTest {
             notDeletedEncounterUuid to null
         )
   }
+
+  @Test
+  fun migration_from_52_to_53_should_add_house_address_and_zone_columns() {
+    val db_52 = helper.createDatabase(52)
+    val selectAllPatientAddressesQuery = "SELECT * FROM PatientAddress"
+    db_52.query(selectAllPatientAddressesQuery).use {
+      assertThat(it.columnCount)
+          .isEqualTo(8)
+    }
+
+    val db_53 = helper.migrateTo(53)
+    db_53.query(selectAllPatientAddressesQuery).use {
```

Comment:
This assertion does not verify that the other columns were not modified as part of this migration. We have an extension function `SupportSQLiteDatabase.assertColumns` which requires passing in all the columns that you expect the table to have. Use that instead?

Reply:
Done.